src/algebra/euclidean_domain/basic.lean

```diff
@@ -84,6 +84,9 @@ begin
   rw [mul_div_cancel_left _ hz, mul_left_comm, mul_div_cancel_left _ hz]
 end
 
+protected lemma mul_div_cancel' (hb : b ≠ 0) (hab : b ∣ a) : b * (a / b) = a :=
```

Comment:
Is there a reason this is primed but `mul_div_cancel_left` is not?

Reply:
This corresponds to `mul_div_cancel'`.